extension-style/src/main/java/com/mapbox/maps/extension/style/image/NinePatchImage.kt

```diff
@@ -0,0 +1,31 @@
+package com.mapbox.maps.extension.style.image
+
+import com.mapbox.maps.Image
+import com.mapbox.maps.ImageContent
+import com.mapbox.maps.ImageStretches
+
+/**
+ * Data class describing 9-patch bitmap.
+ */
+data class NinePatchImage(
+  /**
+   * Bitmap itself.
+   */
+  val internalImage: Image,
+  /**
+   * An array of two-element arrays, consisting of two numbers that represent
+   * the from position and the to position of areas that can be stretched horizontally.
+   */
+  val stretchX: List<ImageStretches>,
+  /**
+   * An array of two-element arrays, consisting of two numbers that represent
+   * the from position and the to position of areas that can be stretched vertically.
+   */
+  val stretchY: List<ImageStretches>,
+  /**
+   * An array of four numbers, with the first two specifying the left, top
+   * corner, and the last two specifying the right, bottom corner. If present, and if the
+   * icon uses icon-text-fit, the symbol's text will be fit inside the content box.
+   */
```

Comment:
nit: incorrect documentation

Reply:
why is it incorrect? seems correct to me (taken from similar docs for `ImageContent` in other places).

Reply:
BTW to me naming of the class itself `ImageContent` seem also very weird. Smth like `ContentPadding` seem much more natural

Reply:
Well, it's generated so we use what we're given...

Reply:
From the documentation of ImageContent:

```
/**
 * Describes the image content, e.g. where text can be fit into an image.
 *
 * When sizing icons with icon-text-fit, the icon size will be adjusted so that the this content box fits exactly around the text.
 */
```

Would be helpful to include these description if this file is not generated..

Also if the documentation sounds confusing I guess we can raise ticket in native to fix it 😄